charts/kueue/templates/visibility/flowschema.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if include "kueue.isFeatureGateEnabled" (dict "List" .Values.controllerManager.featureGates "Feature" "VisibilityOnDemand") }}
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
```

Comment:
I yet need some clarification about the plan here. IIUC when 1.32.0 is released then we use the v1 version in Kueue? 

This will be in about 4 months, so we will be releasing Kueue 0.10 around that time. I'm wondering this can be problematic to Kueue users who want to use latest Kueue (0.10), but cannot upgrade their Kubernetes cluster, for any reason, for example they may have a  policy to only use k8s versions from the STABLE channel (which is lagging a couple of months). How will such users be able to install the latest Kueue on their clusters?

I'm wondering if it is not safer to just put the P&F config (in both versions) in the docs and let the users install the version corresponding to they k8s version (similarly as we did for the visibility-api.yaml). Then, we can embed the P&F config when all versions supported by Kueue support P&F v1.

WDYT @mbobrovskyi @tenzen-y ?

Reply:
My question is which GKE release channel would you like to support in upstream Kueue?
Actually, the GKE provides the following release channels: https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels#channels

My answer depends on your answer.

Reply:
> My question is which GKE release channel would you like to support in upstream Kueue?

I think we need to support all, including Rapid, to enable testing of early versions of Kubernetes. This might be important for new features like MultiKueue (managedBy in Job in particular in 1.32)

Reply:
I discussed the supported GKE channel with @mimowo offline.

In conclusion, we agreed to support the Rapid, Regular, and Stable channels for now (not including Extended).

> I'm wondering if it is not safer to just put the P&F config (in both versions) in the docs

@mimowo could we provide multiple versions of P&F (v1beta3 and v1) for each K8s versions in `/config` directory, and we mention the installation way for each K8s version in https://kueue.sigs.k8s.io/docs/installation/?

This is helpful for the kustomize users since admins can just import the P&F by remote build like:

```yaml
[...]
resources:
- https://github.com/kubernetes-sigs/kueue/cofig/default?ref=v0.8.1
- https://github.com/kubernetes-sigs/kueue/cofig/components/apf/1.28?ref=v0.8.1
[...]
```

Reply:
After we stop the 1.28 support, we can combine the v1 APF manifests in the components directory.

Reply:
Hm, so let me summarize my thinking and check if this is aligned with yours:
1. keeping the P&F configs separate from the installation manifests
2. updating the documentation to provide example configs for v1beta3 and v1 (they could be in config, but maybe also where other example yamls are is better)
3. merge the manifest for v1 into the installation manifests when the last k8s version (1.28) not supporting v1 is EOL (we can add it to the graduation criteria for GA or second Beta)

Reply:
I agree with 1 and 3.

Regarding 2, Does this mean that we will put the APF manifests under the example directory?
Could we take an approach similar to Prometheus installation during the v1.28 is supported?

https://github.com/kubernetes-sigs/kueue/blob/c8353eafd1fff84d569b2937ff4fd85d5e7ebf3b/config/prometheus/kustomization.yaml

After we stop supporting v1.28, we can bundle the APF manifests to default installation manifests as a default.

Reply:
I see, I'm ok to place the examples in the config directory so that they can be used by kustomization.yaml.

Reply:
@mbobrovskyi let's update the KEP with the latest decisions

Reply:
Thanks.

Reply:
Done